JSDemos/Demos/TreeList/UsingHeaderFilter/description.md

```diff
@@ -1 +1,7 @@
-This demo shows how to&nbsp;use the header filter for filtering data. The header filter contains a&nbsp;list of&nbsp;unique column values that you can select. To&nbsp;open this list, click the filter icon in&nbsp;a&nbsp;column header. Once you select a&nbsp;value from this list and click&nbsp;OK, the filter is&nbsp;applied, and the TreeList displays the full hierarchy down to&nbsp;matching rows, but not further.
\ No newline at end of file
+A header filter allows users to select column values from a predefined set and use the selection to filter rows. To enable this behavior, set **headerFilter**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/#visible) to **true**. This feature can be combined with the [filter row](https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeList/UsingFilterRow).
```

Comment:
Once again - what is the reason to give a link to the demo?

```suggestion
A header filter allows users to select column values from a predefined set and use the selected values to filter rows. To enable this behavior, set **headerFilter**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/#visible) to **true**. This feature can be combined with the [filter row](https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeList/UsingFilterRow).
```

Reply:
To give a reference to a related functionality.